README.md

```diff
@@ -59,12 +59,20 @@ sbt test
 
 ## Integration Testing
 
+Start an Elasticsearch server in a docker container:
```

Comment:
As discussed on Slack, starting the docker container seems to fail when docker memory is low. We have seen this happen both when starting up the container (where the container simply exits immediately with no error messages or logs) and in the middle of adding documents to the index. We might want to add a comment here about upping ram if starting the ES container fails